library/core/src/iter/traits/iterator.rs

```diff
@@ -2736,6 +2736,7 @@ pub trait Iterator {
     ///
     /// assert_eq!(iter.next(), None);
     /// ```
+    #[doc(alias = "reverse")]
     #[inline]
     #[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
```suggestion
    #[inline]
    #[doc(alias = "reverse")]
    #[stable(feature = "rust1", since = "1.0.0")]
```

Reply:
I haven't seen anything indicating length-sorting in any of the contribution documentation, and I also haven't seen examples of that in the compiler source. From what I can tell, order is generally (in order of importance):
- Semantically significant ordering (if one attribute affects another)
- Grouping related attributes (e.g. multiple stability-related attributes)
- More-or-less alphabetical.

This ordering seems consistent with that, to me.

Is there some policy I'm unaware of?

Reply:
@joshtriplett I don't think there is a policy, you would know better than me if there was :) I just thought it looked better that way.

Reply:
I just looked one or two examples elsewhere and just did that but I can also change it to that I guess :)